src/apps/media/src/app/components/MediaCreateGroupModal.js

```diff
@@ -45,7 +45,7 @@ export function MediaCreateGroupModal(props) {
           <Button
             variant="contained"
             color="success"
-            onClick={handleCreateGroup}
+            type="save"
```

Comment:
Why is this being added?

Reply:
This was a typo, my bad. Interestingly, it did actually fix the issue tho.